CityOfDetroit/bloom
# Gracefully handle unknowns in filters in backend

## Issue

- Addresses #232 
## Description

Allow filters to include unknown entires for incomplete data in the backend.

## Type of change


## How Can This Be Tested/Reviewed?

Visit localhost:3100 for backend.  Test following URLs to see data.
Toggle between True and False at the end:
http://localhost:3100/listings?filter[0][$comparison]==&filter[0][minRent]=5&filter[0][$include_nulls]=false
http://localhost:3100/listings?filter[0][$comparison]==&filter[0][minRent]=5&filter[0][$include_nulls]=true
http://localhost:3100/listings?filter[0][$comparison]=NA&filter[0][ami]=60&filter[0][$include_nulls]=false
http://localhost:3100/listings?filter[0][$comparison]=NA&filter[0][ami]=60&filter[0][$include_nulls]=true

Additional testing has been added to listing.service.spec.ts.


## Checklist:

- [] I have commented my code, particularly in hard-to-understand areas
